src/is-yes.js

```diff
@@ -0,0 +1,12 @@
+function isYes(text) {
+    text = text.toLowerCase();
+    if (text === 'y') {
+        return true;
+    } else if (text === 'yes') {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+export default isYes;
```

Comment:
Eslint wants you to include one return after your final line. That's what these errors are about.